.github/workflows/closed-issue-reply.yaml

```diff
@@ -2,8 +2,12 @@ name: Closed Issue Message
 on:
   issues:
     types: [closed]
+permissions: {}
```

Comment:
Is this required?

Reply:
The idea is to have restricted default permissions in case someone adds a new job later, but it is optional.